prep-javascript-functions/main.js

```diff
@@ -0,0 +1,58 @@
+function sum(num1, num2) {
+  return num1 + num2;
+}
+
+var sum = sum(2, 2)
+console.log('addTwoNumbersResult = ', sum)
+
+function convertHoursToMinutes(hours) {
+  return hours * 60
+}
+
+var convertHoursToMinutes = convertHoursToMinutes (2)
+console.log('convertHoursToMinutesResult = ', convertHoursToMinutes)
+
+function getGreeting(name) {
+  return "hello " + name + "!"
+}
+
+var getGreeting = getGreeting ("world")
+console.log('helloWorld!Result = ', getGreeting)
+
+function addAndMultiplyBy5(num1, num2) {
+  var num1 = 10
+  var num2 = 5
```

Comment:
You don't want to put these variables here, or else your function won't work for any other numbers except `10` and `5` 😂